CHANGELOG.md

```diff
@@ -3,9 +3,16 @@
 # djinni-intellij-plugin Changelog
 
 ## [Unreleased]
+- nothing
+
+## [0.10.3]
```

Comment:
No, thats wrong 🙈 

1. I doupt that only CLion is affected? Not sure
2. The current changes have to be put into the `[Unreleased]` Block:

```
# djinni-intellij-plugin Changelog

[Unreleased]
- set pluginUntilBuild = 203.* to work with latest IntelliJ Platform

### [0.10.2]
- interfaces can now have `const` methods

...
```

Reply:
should be fixed with the latest commit

Reply:
> I doupt that only CLion is affected? Not sure

Can you check that? My understanding is that the whole IntelliJ Platform got an update to 203, not just CLion?

Reply:
they are all shown as 203.*  in my tool box, yes

Reply:
except, if course, Android Studio, this has somehow different version ...